Use Requires.private pkg-config variable instead of Requires
for internal libspice-server libraries.

--- spice.orig/spice-server.pc.in
+++ spice/spice-server.pc.in
@@ -7,7 +7,7 @@
 Description: SPICE server library
 Version: @VERSION@
 
-Requires: @SPICE_REQUIRES@
+Requires.private: @SPICE_REQUIRES@
 Libs: -L${libdir} -lspice-server
 Libs.private: @SPICE_NONPKGCONFIG_LIBS@
 Cflags: -I${includedir}/spice-server
